### PR TITLE
Fix TS warnings

### PR DESCRIPTION
**Pull request by pittst3r**

### CODE REVIEW

File: src/config/module-map.ts [hunk 1]
```diff
@@ -4,4 +4,6 @@
  *  contents.
  */
 
-export default moduleMap = {};
```
Comment (locks) [inline src/config/module-map.ts:7]: what's the error here?

Comment (pittst3r) [reply src/config/module-map.ts:7]: @locks `moduleMap` needs to be declared. I get `Cannot find name 'moduleMap'.` in VSCode (which pains me to admit being a Vim user).

File: src/main.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import config from './config/environment';
 import moduleMap from './config/module-map';
 
 const resolverConfiguration: ResolverConfiguration = {
-  app: { rootName: config.modulePrefix },
+  app: { name: config.modulePrefix, rootName: config.modulePrefix },
```
Comment (dgeb) [inline src/main.ts:7]: Ah yes, we need to derive `rootName` from `name` (if none is given) in glimmer-resolver. But this more explicit form is needed for now. 👍

**Comment by dgeb** [comment]
Thanks @robbiepitts!

**Pull request merged**